assets/src/dashboard/components/pageStructure/index.js

```diff
@@ -227,8 +227,11 @@ export function LeftRail() {
         <AppInfo>
           {__('\u00A9 2020 Google', 'web-stories')}
           <br />
-          {__('Version', 'web-stories')}&nbsp;
-          {version}
+          {sprintf(
+            /* translators: %s: editor version. */
+            __('Version %s', 'web-stories'),
+            version
+          )}
```

Comment:
According to the Percy screenshots `version` is undefined in the karma tests. Mind adding it so that it's available there?

Otherwise you'll see "Version undefined"